poppy/provider/akamai/background_jobs/check_cert_status_and_update/check_cert_status_and_update_tasks.py

```diff
@@ -28,6 +28,90 @@
 conf(project='poppy', prog='poppy', args=[])
 
 
+class PropertyGetHostnamesTask(task.Task):
+    default_provides = 'latest_version_hosts'
+
+    def __init__(self):
+        super(PropertyGetHostnamesTask, self).__init__()
+        service_controller, self.providers = \
+            memoized_controllers.task_controllers('poppy', 'providers')
+        self.akamai_driver = self.providers['akamai'].obj
+        self.sc = self.akamai_driver.service_controller
+        self.akamai_conf = self.akamai_driver.akamai_conf
+
+        self.existing_hosts = []
+
+    def execute(self, property_spec):
+        """Get latest Akamai property, get its hostnames"""
+        self.property_id = self.akamai_driver.papi_property_id(property_spec)
+
+        LOG.info('Starting to get latest version for property: %s'
```

Comment:
Please consider .format way for first string {0} for second {1} and so on...

Reply:
Yes. This is taken care now